web-wallet/src/lib/stores/stores.d.ts

```diff
@@ -63,10 +62,20 @@ type NetworkStoreServices = {
     options?: NetworkSyncerOptions
   ) => Promise<import("$lib/vendor/w3sper.js/src/mod").AddressSyncer>;
   getCurrentBlockHeight: () => Promise<bigint>;
+  getNetworkName: () => Promise<NetworkName>;
+  getNodeInfo: () => Promise<void | NodeInfo | null>;
 };
 
 type NetworkStore = Readable<NetworkStoreContent> & NetworkStoreServices;
 
+type NodeInfo = {
+  bootstrapping_nodes: Array<string>;
+  chainId: number | null | undefined;
```

Comment:
Seems strange that it can be both `null` and `undefined`.
Actually shouldn't be always a `number`?

Reply:
This is taken from the explorer type definition which works by fetching data from the node, so it might be possible to change it here, however, `number | null` comes from the possible responses from the `node.info` call (unless that changed recently).